### PR TITLE
update icon: awk (original)

### PR DESCRIPTION
**Pull request by Finii**
[why]
There are only wordmark icons for awk.

[how]
Just isolate the bird from the wordmark icons and put it into one dedicated plain icon.

image

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features

Did a bit of research; the wordmark icon is obviously a scan of the 'original' awk book title. Later books all feature a similar but slightly different bird. But it seems this type of bird has become to represent awk.

image
_'Original'_

image
_OReilly version_

image
_Second edition version as photo_

**This PR closes NONE**

## Notes

@Snailedlt as per https://github.com/ryanoasis/nerd-fonts/pull/1691#issuecomment-2313090871

_Edit: Add image of the new icon itself_ :laughing:

### CODE REVIEW

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Good idea using the bird as the non-wordmark version. Just some minor things that need to be fixed before this can be merged.

Also same issue here as before, make sure the base branch is `develop`

File: icons/awk/awk-plain.svg
Comment (Finii) [reply icons/awk/awk-plain.svg:1]: Sorry I am not familiar with the naming rules here and the alias creation. 😬 

About SVGO ... As I took the SVG from inside this repo I thought it must have been optimized already and running the optimizer multiple times degrades outlines and should be avoided, right?

And then there is this sentense [here](https://github.com/devicons/devicon/wiki/SVG-Standards):
>You no longer need to convert classes to fill. We now use SVGO to ensure all IDs and classes are unique.

From this I assumed SVGO is part of the CI.

Comment (Snailedlt) [reply icons/awk/awk-plain.svg:1]: @Finii No worries at all. It's not very clear, and the wiki could use some updating.

A lot of the older icons in this repo are not optimized. Running it through SVGOMG should pose no issues, as it will just stop optimizing the icon at some point. When optimizing using SVGOMG you can use this guide: https://github.com/devicons/devicon/issues/1306#issuecomment-1296015861

The developer who made the optimization CI is not working on devicons anymore. I'm not sure how it works, but I think it's only applied before making the font version, so the SVGs still need optimization.

**Review by Snailedlt — APPROVED** [review]
Great work!
Thank you for the contribution!

We just need one more approval before this can be merged into the `develop` branch

File: icons/awk/awk-plain.svg
Comment (Finii) [reply icons/awk/awk-plain.svg:1]: Hmm, at NerdFonts the post-PR-created CI adds commits with optimized SVGs (if the optimized version is a certain amount of smaller than the originally PRed one). [1]

I can have a look at your CI if there is no one 'responsible'. Otoh, some contributors don't get it if the CI adds commits to their branch ;-)
Whatever you do, it's wrong ;)

[1] https://github.com/ryanoasis/nerd-fonts/blob/master/.github/workflows/packsvgs.yml#L23-L40

**Comment by Finii** [comment]
There is one CI issue, something with the PR title I gleam from the error snippet :grimacing: 
Fini - clueless

File: icons/awk/awk-plain.svg
Comment (Snailedlt) [reply icons/awk/awk-plain.svg:1]: > I can have a look at your CI if there is no one 'responsible'.

That would be awesome! We need someone to work on automation in general too if you're interested. We have an issue about creating a CI for optimization. Here it is: https://github.com/devicons/devicon/issues/1547

There are also a ton of other automation and workflow issues open in case you're interested: https://github.com/devicons/devicon/issues?q=is%3Aopen+is%3Aissue+label%3Adevops

**Comment by Snailedlt** [comment]
> There is one CI issue, something with the PR title I gleam from the error snippet 

Ahh yes, the title was incorrect. I fixed it now. If you take a look at the PR template when you create a PR you'll see what the title should look like depending on which kind of PR it is :)

**Review by canaleal — APPROVED** [review]

**Pull request merged**
